sys/include/tasklet.h

```diff
@@ -162,92 +165,92 @@ extern "C" {
 #endif
 
 /**
- * @brief   Default priority of the interrupt handler thread
+ * @brief   Default priority of the tasklets thread
  *
- * The priority of the interrupt handler thread has to be high enough that all
- * pending interrupts are handled before other threads are executed.
+ * The priority of the taskletthread has to be high enough that all
+ * pending tasklets are handled before other threads are executed.
  */
-#ifndef IRQ_HANDLER_PRIO
-#define IRQ_HANDLER_PRIO    0
+#ifndef TASKLET_HANDLER_PRIO
+#define TASKLET_HANDLER_PRIO    0
 #endif
 
 /**
- * @brief   Interrupt handling function prototype
+ * @brief   Tasklet handling function prototype
  *
  * Defines the prototype of the function that is registered together
- * with an interrupt event and to be called when the interrupt is handled.
+ * with a taskletand to be called when the tasklet is handled.
  */
-typedef void (*irq_isr_t)(void *ctx);
+typedef void (*tasklet_task_t)(void *ctx);
```

Comment:
I'm not sure if "task" is the right term. To me, this is the signature of a function that is executing the actual task. But the function itself is not the task. Maybe a different term would be clearer?